src/modules/previewpane/UnitTests-SvgPreviewHandler/SvgPreviewControlTests.cs

```diff
@@ -89,13 +85,107 @@ public void SvgPreviewControl_ShouldSetScrollBarsEnabledProperty_WhenDoPreviewCa
         {
             // Arrange
             var svgPreviewControl = new SvgPreviewControl();
-            var mockStream = new Mock<IStream>();
 
             // Act
-            svgPreviewControl.DoPreview(mockStream.Object);
+            svgPreviewControl.DoPreview(GetMockStream("<svg></svg>"));
 
             // Assert
             Assert.AreEqual(((WebBrowser)svgPreviewControl.Controls[0]).ScrollBarsEnabled, true);
         }
+
+        [TestMethod]
+        public void SvgPreviewControl_ShouldAddTextBox_IfBlockedElementsArePresent()
+        {
+            // Arrange
+            var svgPreviewControl = new SvgPreviewControl();
+
+            // Act
+            svgPreviewControl.DoPreview(GetMockStream("<svg>\r\n <script> valid </script>\r\n </svg>"));

```

Comment:
Just to make this test more robust it would be better to add multiple <script> tags

Reply:
Added a Test for validating multiple blocked tags in `SvgPreviewHandlerHelperTests.cs`. This tests is to test add a text box if there is a blocked element present.